src/imagery/i.saocom/i.saocom.import/i.saocom.import.py

```diff
@@ -0,0 +1,279 @@
+                bands[pol]["array"] = banda.read()
+                bands[pol]["metadata"] = banda.meta
+                gcps = banda.get_gcps()
+    return bands, gcps, dataset_pols
+
+
+def read_bands_folder(data, pols):
+    file_list = [
+        img
+        for img in os.listdir(os.path.join(data, "Data"))
+        if not img.endswith(".xml")
+    ]
+    bands = {}
+    gcps = tuple()
+    dataset_pols = []
+    for l in file_list:
+        pol = l.split("-")[6]
+        dataset_pols.append(pol)
+        if pol in pols:
+            bands[pol] = {}
+            filename = os.path.join(data, "Data", l)
+            with rasterio.open(filename) as banda:
+                bands[pol]["array"] = banda.read()
+                bands[pol]["metadata"] = banda.meta
+                gcps = banda.get_gcps()
+    return bands, gcps, dataset_pols
+
+
+def save_bands(bands, basename):
+    for band in bands:
+        # Change the band metadata:
+        # Data type must be changed from complex to float
+        bands[band]["metadata"]["dtype"] = np.float32
+        # The Y-resolution must be set to negative, otherwise GRASS will interpret the map is flipped
+        geoTs = bands[band]["metadata"]["transform"]
+        bands[band]["metadata"]["transform"] = Affine(
+            geoTs[0], geoTs[1], geoTs[2], geoTs[3], -geoTs[4], geoTs[5]
+        )
+        outputfn_r = f"{basename}_{band}_real.tif"
+        outputfn_i = f"{basename}_{band}_imag.tif"
+        with rasterio.open(outputfn_r, "w", **bands[band]["metadata"]) as dst:
+            dst.write(bands[band]["array"].real)
+        with rasterio.open(outputfn_i, "w", **bands[band]["metadata"]) as dst:
+            dst.write(bands[band]["array"].imag)
+
+
+def main():
+    # xemt = options["xemt"]
+    data = options["data"]
+    zip_v = options["is_zip"]
+    basename = options["basename"]
+    pols = options["pols"]
+    multilook = options["multilook"]
+
+    if zip_v == "yes":
+        bands, gcps, dataset_pols = read_bands_zip(data, pols)
+    else:
+        bands, gcps, dataset_pols = read_bands_folder(data, pols)
+
+    if len(bands) == 0 or len(gcps) == 0:
+        grass.fatal(
+            _(
+                f"None of the specified polarizations were found in the dataset \n Please try one of the folowing: {dataset_pols}"
+            )
+        )
+        # pass
+
+    else:
+        # Create a dataframe containing GCP information
+        cols = gcps[0][0].asdict().keys()
+        df = pd.DataFrame(columns=cols)
+        for i, gcp in enumerate(gcps[0]):
+            df1 = pd.DataFrame(gcp.asdict(), index=[i])
+            df = pd.concat((df, df1))
+
+        if multilook[0] != 1 or multilook[2] != 1:
+            # ~ print(f'Appying ML factor: {multilook}')
+            grass.message(_(f"Appying ML factor: {multilook}"))
+            for band in bands:
+                dim = bands[band]["array"].shape
+                # ~ print('Original shape ', bands[band]['array'].shape)
```

Comment:
```suggestion

```

Reply:
please remove the print() calls below as they are covered by grass messages already